apps/admin/src/containers/Purchase/index.tsx

```diff
@@ -72,7 +71,7 @@ export const Purchase = () => {
         <Form.Button onClick={() => setConfirm(true)}>SEND</Form.Button>
       </Form>
       <Confirm open={confirm} onCancel={() => setConfirm(false)} onConfirm={onConfirm} />
-      <ModalOperation isOpen={isProcessing} header={messages.mintingHeader} progressMessage={messages.mintingInProgress} messageValues={{ step }} />
+      <ModalOperation isOpen={isProcessing} header={messages.mintingHeader} progressMessage={messages.mintingInProgress} />
```

Comment:
Passing the values through as a separate entry could cause issues when the component re-rendered too quickly (ie after setting message, but before value, or vice-versa).

By merging message & value, we eliminate the possibility that the we might render during a transition state